fl4health/parameter_exchange/layer_exchanger.py

```diff
@@ -92,88 +91,76 @@ def pull_parameters(self, parameters: NDArrays, model: nn.Module, config: Option
         model.load_state_dict(current_state, strict=True)
 
 
-class NormDriftParameterExchanger(ParameterExchangerWithPacking[List[str]]):
+class DynamicLayerExchanger(PartialParameterExchanger[List[str]]):
     def __init__(
         self,
+        layer_selection_function: Callable[..., Tuple[NDArrays, List[str]]],
         norm_threshold: float,
         exchange_percentage: float = 0.1,
         normalize: bool = True,
         filter_by_percentage: bool = True,
+        select_drift_more: bool = True,
     ) -> None:
         """
```

Comment:
Small documentation ask. It may be helpful to have some documentation on the arguments just clarify what these are in this class and downstream function that leverage them.